zipline/finance/order.py

```diff
@@ -37,8 +41,8 @@
 
 
 class Order(object):
-    def __init__(self, dt, sid, amount, stop=None, limit=None, filled=0,
-                 commission=None, id=None):
+    def __init__(self, dt, sid, amount, cancel_policy=None, stop=None,
```

Comment:
IMO this shouldn't be a default if the expectation is that every `Order` instance will have this attribute.  c.f. @richafrank's comment below.